src/core/Form/RadioButton/RadioButton.md

```diff
@@ -25,90 +64,102 @@ const handleChange = (event) => {
   setSelectedValue(event.target.value);
```

Comment:
This isn't used in this example anymore it seems. So should either be removed or have onChange use this one.

Reply:
Good catch, will fix!

Reply:
Fixed.